### DISCUSSION TITLE
User Specific  Multiple Directory

### DISCUSSION THREAD

**Discussion by kaynnan** [prasathmani/tinyfilemanager#778] [General]
Is it possible to add multiple paths for that user? For example, user A has access to the AB,CD,DF folder, however, I can't read the AB,CD folders only the DF, is it possible to list the previous ones?

**Comment by prasathmani** [comment]
not possible with current version

**Reply by phihungvohoang** [reply]
Can we use it?
$directories_users = array(
     'user1' => array('Doc', 'user1', 'user2'),
     'user2' => array('Doc', 'user2')
);
